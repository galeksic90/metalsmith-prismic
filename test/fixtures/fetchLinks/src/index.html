---
template: index.hbt
prismic:
  jobOffers:
    query: '[[:d = any(document.type, ["job-offer"])]]'
    arrayFragments: true
    fetchLinks: 'store.name,store.address,product.name'
---
