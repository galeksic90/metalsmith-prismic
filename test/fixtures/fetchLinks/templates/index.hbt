<html>
<body>
<ul>
  {{#each prismic.jobOffers.results}}
    <li>
        {{{data.name.html}}}
        <ul>
          {{#each data.location}}
            <li>
                <span>{{children.name.json.0.text}}</span>
                {{{children.address.html}}}
            </li>
          {{/each}}
        </ul>
    </li>
  {{/each}}
</ul>
</body>
</html>
