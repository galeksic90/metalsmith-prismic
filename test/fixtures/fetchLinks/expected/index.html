<html>
<body>
<ul>
    <li>
        <h1>Art Director</h1>
        <ul>
            <li>
                <span>Paris Saint-Lazare</span>
                <span>56 rue Saint-Lazare</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Store Intern</h1>
        <ul>
            <li>
                <span>New York Fifth Avenue</span>
                <span>767 5th Avenue</span>
            </li>
            <li>
                <span>Tokyo Roppongi Hills</span>
                <span>2-29-1 Dougenzaka</span>
            </li>
            <li>
                <span>London Covent Garden</span>
                <span>1, The Market</span>
            </li>
            <li>
                <span>Paris Saint-Lazare</span>
                <span>56 rue Saint-Lazare</span>
            </li>
            <li>
                <span>Paris Champ-Elysées</span>
                <span>101 avenue des Champs-Élysées</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Content Director</h1>
        <ul>
            <li>
                <span>Paris Saint-Lazare</span>
                <span>56 rue Saint-Lazare</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Ganache Specialist</h1>
        <ul>
            <li>
                <span>Paris Saint-Lazare</span>
                <span>56 rue Saint-Lazare</span>
            </li>
            <li>
                <span>Tokyo Roppongi Hills</span>
                <span>2-29-1 Dougenzaka</span>
            </li>
            <li>
                <span>London Covent Garden</span>
                <span>1, The Market</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Community Manager</h1>
        <ul>
            <li>
                <span>Paris Saint-Lazare</span>
                <span>56 rue Saint-Lazare</span>
            </li>
            <li>
                <span>Tokyo Roppongi Hills</span>
                <span>2-29-1 Dougenzaka</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Oven Instrumentist</h1>
        <ul>
            <li>
                <span>New York Fifth Avenue</span>
                <span>767 5th Avenue</span>
            </li>
            <li>
                <span>Tokyo Roppongi Hills</span>
                <span>2-29-1 Dougenzaka</span>
            </li>
            <li>
                <span>Paris Champ-Elysées</span>
                <span>101 avenue des Champs-Élysées</span>
            </li>
            <li>
                <span>London Covent Garden</span>
                <span>1, The Market</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Pastry Dresser</h1>
        <ul>
            <li>
                <span>Tokyo Roppongi Hills</span>
                <span>2-29-1 Dougenzaka</span>
            </li>
        </ul>
    </li>
    <li>
        <h1>Store Assistant Manager</h1>
        <ul>
            <li>
                <span>Paris Champ-Elysées</span>
                <span>101 avenue des Champs-Élysées</span>
            </li>
            <li>
                <span></span>
                
            </li>
        </ul>
    </li>
    <li>
        <h1>Fruit Expert</h1>
        <ul>
            <li>
                <span>Paris Saint-Lazare</span>
                <span>56 rue Saint-Lazare</span>
            </li>
            <li>
                <span>London Covent Garden</span>
                <span>1, The Market</span>
            </li>
        </ul>
    </li>
</ul>
</body>
</html>
